rust/rsmgp-sys/src/result/mod.rs

```diff
@@ -137,44 +134,72 @@ pub enum MgpError {
     #[snafu(display("Unable to access list value by index."))]
     UnableToAccessListValueByIndex,
 
+    // MAP
+    #[snafu(display("Unable to copy map."))]
+    UnableToCopyMap,
```

Comment:
Why map/list is using `UnableToCopyXXX` while edge/vertex/path uses `UnableToCreateXXXCopy`? Is this intentional?

Reply:
It's not, it's just inconsistent, I'll update everything to the shortest version `UnableToCopyXYZ` :smile: